pg_shard.c

```diff
@@ -108,7 +123,12 @@ static Const * ExtractPartitionValue(Query *query, Var *partitionColumn);
 static bool ExtractFromExpressionWalker(Node *node, List **qualifierList);
 static List * QueryFromList(List *rangeTableList);
 static List * TargetEntryList(List *expressionList);
+static List * TargetEntryVarList(List *targetEntryList);
+static CreateStmt * CreateTemporaryTable(Query *localQuery, Query *distributedQuery,
```

Comment:
This function name is misleading; it doesn't create a table, it creates a create table statement.

Maybe `CreateTemporaryTableStmt`?